pinot-core/src/main/java/org/apache/pinot/core/transport/ServerResponse.java

```diff
@@ -50,6 +51,14 @@ public int getSubmitDelayMs() {
     }
   }
 
+  public int getRequestSentDelayMs() {
```

Comment:
This is not being called anywhere so it won't be used and that's why I suggested in previous comment that you probably don't need to `asyncQueryResponse.markRequestSent(serverRoutingInstance);`

Reply:
If we need this value to be logged in ServerStats like other stats are logged in BaseBrokerRequestHandler, then let's change the toString implementation of ServerResponse since that is what is called by `AsyncQueryResponse.getStats()`

Reply:
I think it's still worthwhile to keep it in ServerResponse, which makes it clearer about the delay from broker to server in the log. I'll add it to the toString method in the next push.